src/hotspot/share/opto/escape.cpp

```diff
@@ -3739,6 +3791,38 @@ void ConnectionGraph::dump(GrowableArray<PointsToNode*>& ptnodes_worklist) {
   }
 }
 
+void ConnectionGraph::print_statistics() {
+  tty->print_cr("No escape: %d", Atomic::load(&_no_escape_counter));
```

Comment:
This is just my suggestion. I would say that we keep each print_statistics() method oneliner.  
It will ease parsers.  in Compile::print_statistics, most of them follow the pattern: phase: counter1, couter2, ...
```  
  tty->print_cr("Peephole: peephole rules applied: %d",  _total_peepholes);
```

Reply:
@navyxliu Fixed! And I included a new example of how it prints above!